helm_deploy/values-dev.yaml

```diff
@@ -19,7 +19,7 @@ env:
   HMPPSAUTH_BASEURL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   INTERVENTIONSUI_BASEURL: "https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk"
   COMMUNITYAPI_BASEURL: "https://community-api-secure.test.delius.probation.hmpps.dsd.io"
-  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_PROBATIONAREACODE: "CRS"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_PROVIDERCODE: "CRS"
```

Comment:
this also needs updating in the template _envs.tpl file

Reply:
Missed this one - have updated